### PR TITLE
docs(single): Updated docs for single with description and example (#…

### PR DESCRIPTION
**Pull request by weitalu**
**Description:**
Updated docs for single with description and example
**Related issue (if exists):**
#4616

### CODE REVIEW

File: src/internal/operators/single.ts [hunk 1]
```diff
@@ -11,8 +11,34 @@ import { Observer, MonoTypeOperatorFunction, TeardownLogic } from '../types';
...
+ * import { single } from 'rxjs/operators';
+ *
+ * const numbers = range(1,5).pipe(single());
+ * numbers.subscribe(x => console.log('never get called'), e => console.log('error'));
+ * // result
+ * // 'error'
+ * ```
+ *
+ * emits 'undefined'
+ * ```javascript
```
Comment (JWO719) [inline src/internal/operators/single.ts]: could you change the examples to ts

Comment (weitalu) [reply src/internal/operators/single.ts]: do you mean that change the word 'javascript' to 'typescript',

or use some feature of typescript in the example like interface or type ?

Comment (dzhavat) [reply src/internal/operators/single.ts]: What @JWO719 meant was changing

```
```javascript
```

to

```
```ts
```

Comment (weitalu) [reply src/internal/operators/single.ts]: Thank you for the hint !

**Pull request merged**